cmds/core/uptime/uptime_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"testing"
+	"time"
+)
+
+var (
+	testTime = time.Date(0001, 1, 15, 5, 35, 49, 0, time.UTC)
+)
+
+func TestUptime(t *testing.T) {
+	var tests = []struct {
+		name   string
+		input  string
+		uptime *time.Time
+		err    string
+	}{
+		{"goodInput", "1229749 1422244", &testTime, ""},
+		{"badDataInput", "string", nil, "Error time: invalid duration strings"},
+		{"emptyDataInput", "", nil, "Error:the contents of proc/uptime we are trying to read are empty"},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			gotUptime, err := uptime(test.input)
+			if err == nil && test.err != "" {
+				t.Errorf("Error was not returned, expected %v", test.err)
+			} else if err != nil && err.Error() != test.err {
+				t.Errorf("Mismatched Error returned, expected \n%v\n got \n%v\n", test.err, err.Error())
+			}
+			if gotUptime == nil && test.uptime != nil {
+				t.Errorf("Error mismatched uptime:expected %v , got <nil>", *test.uptime)
```

Comment:
change all to:
got %v, want nil
or 
got %v, want %v
or
got nil, want %v

same format makes it easier to search

Reply:
Done